Adding REAME file to the Amplication logger module (#3356)
amplication/amplication: packages/amplication-nest-logger-module/README.md

> # Amplication nestjs logger module

Reviewer: ```suggestion
# Amplication NestJS Logger Module
```
This is a heading, and we mention a technology (NestJS). So we need to capitalize it properly.

> Amplication NestJs logger module is a standards wrapper for nest-Winston that it is a wrapper for Winston logger.
> We need this library to ensure that all relevant metadata is included in every log.

Reviewer: ```suggestion
The Amplication NestJS Logger Module is our official logger module. Currently, it is a wrapper around the `nest-winston` package.

All services at Amplication must use this library to ensure consistent logging. This will enable us to trace and observe our systems reliably.
```

We need to make sure we mention technology names accurately.

`NestJs` -> `NestJS
`nest-Winston` -> `nest-winston`

> ## How to consomme

Reviewer: ```suggestion
## How to consume
```
Make sure to run through Grammarly

We did an example in pr [#3285](https://github.com/amplication/amplication/pull/3285) with all the details.
In short, to consomme this package, you need to add it to the [package.json](https://github.com/amplication/amplication/blob/master/packages/amplication-git-pull-request-service/package.json#L23) and the package have three main exports

|  Name | Description  |
| ------------ | ------------ |
| AmplicationLoggerModule  |  The Amplication logger module that wraps the logic and insatiate the nest-Winston module |
| AMPLICATION_LOGGER_PROVIDER  |  The NestJs provider key to consomme the inject the instance |

> | AmplicationLogger  |  An interface that wraps the logger interface from the Winston package |

Reviewer: While this is workable, I think we should aim for something that is clearer and better. Let's spend 10 minutes on this to write this together :)
